ambry-store/src/test/java/com.github.ambry.store/MockId.java

```diff
@@ -23,16 +23,18 @@
 public class MockId extends StoreKey {
 
   private String id;
+  private short version = 3;
```

Comment:
no versions here - the impl need not be the same as `BlobId` - just needs to return `true`/`false` as required for `isAccountContainerMatch()`

Reply:
fixed.